src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -394,6 +394,10 @@ public record Transcript()
   public bool IsOfficialTranscriptRequested { get; set; }
   [Required]
   public bool IsNameUnverified { get; set; }
+  [Required]
```

Comment:
It looks like we don't follow this approach here, but it would be great if we could move the required fields into the constructor, similar to how it's done in the `WorkExperienceReference` record.